docs/rules/no-sequences.md

```diff
@@ -17,7 +17,6 @@ while (a = next(), a && a.length);
 This rule forbids the use of the comma operator, with the following exceptions:
 
 * In the initialization or update portions of a `for` statement.
-* If the expression sequence is explicitly wrapped in parentheses.
```

Comment:
Since `allowInParentheses` defaults to true, this exception still applies when this rule is configured with its defaults. What do you think about indicating that here? Something like "By default, if the expression sequence is explicitly wrapped in parentheses. This exception can be removed with the `allowInParentheses` option."

Reply:
@btmills You are right, I removed that sentence in my first iteration with option's default value set to `false`. Brought it back as you suggested.